styles/stylesheet.css

```diff
@@ -0,0 +1,555 @@
+    display: flex;
+    flex-direction: row;
+    padding: 1em;
+}
+
+.homepage-intro img {
+    width: 100%;
+}
+
+@media screen and (max-width: 500px) {
+    .dog-card-display, .dog-card-display-full {
+        display: grid;
+        grid-template-columns: 1fr;
+        grid-auto-rows: 1fr;
+        margin-bottom: 3em;
+    }
+}
+
+@media screen and (min-width: 500px) and (max-width: 1068px) {
+    .dog-card-display {
+        display: grid;
+        grid-template-columns: repeat(2, 1fr);
+        grid-auto-rows: 1fr;
+        margin-bottom: 3em;
+
+    }
+    .dog-card-display-full {
+        display: grid;
+        grid-template-columns: repeat(3, 1fr);
+        grid-auto-rows: 1fr;
+        margin: 0% 15%;
+
+    }
+}
+
+@media screen and (min-width: 1068px) {
+    .dog-card-display {
+        display: grid;
+        grid-template-columns: repeat(3, 1fr);
+        grid-auto-rows: 1fr;
+        margin-bottom: 3em;
+
+    }
+    .dog-card-display-full {
+        display: grid;
+        grid-template-columns: repeat(4, 1fr);
+        grid-auto-rows: 1fr;
+        margin: 0% 15%;
+
+    }
+}
+
+.dog-card-price-description > span {
+    font-weight: bold;
+}
+
+.dog-card {
+    max-width: 100%;
+    margin: 1em;
+    border-style: solid;
+    border-color: #dad8d2;
+    border-width: 1px;
+    font-size: small;
+}
+
+.dog-card > h2{
+    margin: 0rem 1rem;
+}
+
+.dog-card > p {
+    margin: 0.5rem 1rem;
+}
+
+.dog-card a {
+    display: block;
+    text-align: center;
+    margin: 1rem;
+    padding: .5em;
+    border-radius: 3px;
+    text-decoration: none;
+    font-weight: bold;
+    color: #ffffff;
+    background-color: #86b349;
+}
+
+.dog-card-image {
+    max-width: 100%;
+}
+
+.dog-card-bold {
+    font-weight: bold;
+    margin-right: 0.25em;
+}
+
+.see-more-button {
+    text-align: center;
+    text-decoration: none;
+    background-color: #dad8d2;
+    color: #113c60;
+    font-weight: bold;
+    font-size: large;
+    padding: 1rem 4rem;
+    border-radius: 3px;
+}
+
+@media screen and (max-width: 500px) {
+    .blog-post-previews {
+        display: none;
+    }
+}
+
+@media screen and (min-width: 500px) {
+    .blog-post-previews {
+        grid-column-start: 2;
+    }
+}
+
+@media screen and (max-width: 500px) {
+    .blog-post {
+        display: flex;
+        flex-direction: column;
+    }
+
+    .blog-post-text {
+        display: flex;
+        flex-direction: column;
+        margin: 1em;
+    }
+}
+
+@media screen and (min-width: 500px) {
+    .blog-post {
+        font-size: larger;
+        display: flex;
+        flex-direction: row;
+        margin: 0% 15%;
+        padding: 1em;
+    }
+
+    .blog-post-text {
+        padding: 0em 1em;
+        display: flex;
+        flex-direction: column;
+        max-height: 100%;
+    }
+}
+
+.blog-post-text > h2 {
+    font-size: 1.5em;
+    margin: 1em 0px 0px 0px;
+}
+
+.blog-post-text > p {
+    line-height: 110%;
+}
+
+.blog-post-image {
+    object-fit: contain;
```

Comment:
I would consider adding width:100% so that the images scale as the page shrinks, and do not affect the header.